src/order/order.ts

```diff
@@ -33,6 +33,7 @@ export default interface Order {
     taxes: Tax[];
     taxTotal: number;
     mandateUrl?: string;
+    providerData?: ProviderData;
```

Comment:
If this data is specific to a payment, shouldn't it be within a `OrderPayments` object in `payments` rather than at the root of the order?

Reply:
Same with `mandateUrl` above

The usage in https://github.com/bigcommerce/checkout-js/pull/674/files makes it look like `mandateUrl` and `ProviderData` are mutually exclusive. 

If so, would this better represent our business logic?

```
export type OrderPayments = Array<GatewayOrderPayment | GiftCertificateOrderPayment | DirectDebitOrderPayment>;

interface DirectDebitOrderPayment extends OrderPayment {
    mandate: { url: string } | { id: string };
}
```